examples/BeaconDemo/BeaconDemo.ino

```diff
@@ -0,0 +1,42 @@
+#include <SPI.h>
+#include <SPBTLE_RF.h>
+#include <beacon_service.h>
+
+/* Configure SPI3:
+  MOSI: PC12
```

Comment:
Specific to disco IOT ? rename example or add comment on top of the scketch to specify it

Reply:
In addition, in the comments, it would be good to add a description of the example. What does the example, how to test, how to customize...
Most of the information is in the readme, but the user will probably open only the sample...
Then, for example, if we want to switch from UID to URL mode, it is not obvious.